build-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eu
+
+pushd docker
+docker build -t stowtest .
+popd
```

Comment:
This `popd` is redundant, because the script will run in a different non-interactive shell process to the interactive one which launched it.